libs/core/src/lib/rating-indicator/components/rating-indicator.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="fd-rating-indicator__container">
+    <!-- Zero value if we don't have provided value from user, `ratingAverage` or `ratings` -->
+    <ng-container *ngTemplateOutlet="zeroRate"></ng-container>
+    <!-- Available rate values -->
+    <ng-container *ngFor="let rate of _rates; index as i; trackBy: trackByFn">
+        <input
+            [attr.aria-label]="(i + 1) + '-star'"
+            type="radio"
+            class="fd-rating-indicator__input"
+            [disabled]="disabled"
+            [id]="rate.id"
+            name="rating-{{ viewRatingUID }}"
+            [value]="rate.value"
+            [checked]="viewValue === rate.value"
+            (focus)="onSelect(rate.value)" />
+        <label class="fd-rating-indicator__label" [for]="rate.id">
+            <i *ngIf="ratedIcon" class="rated" [ngClass]="unratedIcon"></i>
+            <i *ngIf="unratedIcon" class="unrated" [ngClass]="ratedIcon"></i>
+        </label>
+    </ng-container>
+</div>
+<!-- Popover if displaySumPopover is true and we have ratings object -->
+<fd-popover
```

Comment:
I think we should switch to [this popover markup](https://fundamental-ngx.netlify.app/#/core/popover#trigger) to make the code more readable.

Reply:
Trigger directive hasn't any options for custom popover (triggers, noArrow, additionalBodyClass etc.)

Reply:
@mykolavorobiov Yeah, you still put all of the popover configuration on `fd-popover` element.

Reply:
@mykolavorobiov Yeah, you still put all of the popover configuration on fd-popover element.